### ISSUE TITLE
The loss is Nan

### ISSUE THREAD

**Issue by FlyEgle** [PeizeSun/OneNet#19]
hello, i easy modify the detectron2 code to simpliy for self train,  and work for the face detection on widerface
But, i find the giou loss may nan which i have use the lr with [0.01, 0.001, 0.0001, 0.00001, 0.000001] with ADAMW.

I want know are you have the nan occur or do grad clip for it ? 
thx

**Comment by csliuchang** [comment]
i also have a problem with loss, i use onenet for one class detection, but loss_ce is not decay, and AP is very lower than 80 class detection is round 0.01. have u solve this question? could u give me some advice?

**Comment by FlyEgle** [comment]
I use the ADAMW with 5e-5 lr for training, i have not final the train.The iou loss and L1 loss not decay well, but class loss have a little decay.
 
![image](https://user-images.githubusercontent.com/26165705/107305154-658c7880-6abd-11eb-93b9-b0edf05bb815.png)

**Comment by csliuchang** [comment]
当onenet num_classes为1时候，输出的logits的负值特别大，导致scoremap归0，无法导出结果，还不知道为什么。loss_ce开始很大然后很小几乎等于0
